test/utils/test-utils.js

```diff
@@ -292,6 +292,15 @@ const runServe = (args, testPath) => {
     });
 };
 
+const hyphenToUpperCase = (name) => {
```

Comment:
Seems duplicate, possible to import it?

Reply:
It's defined as a method in `GroupHelper` util,

Reply:
possible to move it to utils package? any downsides?

Reply:
Ye we can move, we just have to import the `GroupHelper` util and create it's new instance. I will make the necessary changes.

Reply:
but this may cause noise in this PR, can we do this in a future PR ?

Reply:
Um okay